cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -686,11 +690,26 @@ pAddCommitteeColdVerificationKeyFile =
     ]
 
 ---
-pRemoveCommitteeColdVerificationKeyOrHashOrFile :: Parser (VerificationKeyOrHashOrFile CommitteeColdKey)
+pRemoveCommitteeColdVerificationKeyOrHashOrFile :: Parser (VerificationKeyOrHashOrFileOrScriptHash CommitteeColdKey)
```

Comment:
If being nitpicky, this function should be named `pRemoveCommitteeColdVerificationKeyOrHashOrFileOrScriptHash` (note the additional suffix).

But probably this is overkill, since this is also indicated in the return type. So maybe just rename to `pRemoveCommitteeColdVerificationKeySource`?

The same remark applies to `pAddCommitteeColdVerificationKeyOrHashOrFile`.

Reply:
Thanks, good idea. I think we can go with source since it's a single-use parser and it would define what's happening in an unique way.